examples/microwave-oven-app/linux/third_party/connectedhomeip

```diff
@@ -0,0 +1 @@
+../../../../
```

Comment:
Why is this "third_party" directory needed?

Reply:
This is needed in "./scripts/build/build_examples.py" while compiling app.

> Why is this "third_party" directory needed?